### PR TITLE
Switch from scss to tailwind

### PR DESCRIPTION
**Pull request by foxy-methoxy**

### CODE REVIEW

File: src/components/Products/ProductsList.vue [hunk 1]
```diff
@@ -1,27 +1,22 @@
...
+                        :price="item.price"
+                        :imgUrl="item.imgUrl"
+                        :rating="item.rating"
+                        @click="addToCart(item)">
     </products-list-item>
   </div>
 
   <div v-else class="empty-product-list">
-	  <strong>Product list is empty</strong>
+    <strong>Product list is empty</strong>
```
Comment (kevinikos) [inline src/components/Products/ProductsList.vue]: use translation instead of plain text

File: src/components/Products/ProductsListItem.vue [hunk 2]
```diff
@@ -1,111 +1,81 @@
...
-        :style="{ backgroundImage: 'url(' + imgUrl + ')' }"
-    ></div>
-    <div class="product-list-item__description">
-      <span
-          class="product-list-item__description__title"
-          v-if="name.length < 20"
-      >{{ name }}
+	<div class="product-list-item">
+		<div class="product-list-item__picture"
+         :style="{ backgroundImage: 'url(' + imgUrl + ')' }"></div>
```
Comment (kevinikos) [inline src/components/Products/ProductsListItem.vue]: align attributes and add line break before next html element

File: src/components/Products/ProductsListItem.vue [hunk 3]
```diff
@@ -1,111 +1,81 @@
...
-    <div class="product-list-item__rating">
-      <IconStar class="product-list-item__star"/>
-      <span>{{ rating }} </span>
-    </div>
-  </div>
+			<span class="product-list-item__description__price">${{ price }}</span>
+		</div>
+		<div class="product-list-item__rating">
+			<IconStar class="product-list-item__star" />
+			<span>{{ rating }} </span>
```
Comment (kevinikos) [inline src/components/Products/ProductsListItem.vue]: remove whitespace between text and closing `span` tag

File: src/components/Products/ProductsListItem.vue [hunk 4]
```diff
@@ -1,111 +1,81 @@
...
+	name: "ProductsListItem",
+	props: {
+		id: String,
+		name: String,
+		type: String,
+		price: Number,
+		imgUrl: String,
+		rating: Number
+	},
+	computed: {
```
Comment (kevinikos) [inline src/components/Products/ProductsListItem.vue]: you should move `computed` to `setup` function which is part of `vue-composition-api` feature,
also if you want to use something like `mapGetters` you can check `vue-composition-helpers`
https://github.com/greenpress/vuex-composition-helpers

File: src/components/Products/ProductsListItem.vue [hunk 5]
```diff
@@ -1,111 +1,81 @@
...
+		</div>
+	</div>
 </template>
 
 <script lang="ts">
-import {mapGetters} from "vuex";
+import { mapGetters } from "vuex";
 import IconStar from "@/assets/icons/icon-stars.vue";
 
 export default {
```
Comment (kevinikos) [inline src/components/Products/ProductsListItem.vue]: use `defineComponent({...})` instead of `{...}`

**Review by kevinikos — CHANGES_REQUESTED** [review]

File: src/components/Products/ProductsListItem.vue [hunk 6]
```diff
@@ -1,25 +1,29 @@
 <template>
 	<div class="product-list-item">
 		<div class="product-list-item__picture"
-         :style="{ backgroundImage: 'url(' + imgUrl + ')' }"></div>
+         :style="{ backgroundImage: 'url(' + imgUrl + ')' }">
+    </div>
+
```
Comment (foxy-methoxy) [inline src/components/Products/ProductsListItem.vue]: w webstormie wyswietlaja sie dobrze wciecia?

**Review by entrop- — APPROVED** [review]

**Pull request merged**